pyrokinetics/kinetics/jetto.py

```diff
@@ -156,6 +167,11 @@ def read_from_file(
 
             density_func = UnitSpline(psi_n, density_data)
 
+            if species["species_name"] == "alphas":
```

Comment:
This may be nit picky but do we want to call this `alpha` or `alphas`. We don't use the plural for electrons for example. Its a bit confusing as I guess the plural for helium is helium but I think we should be consistent